docs/community/roadmap.md

```diff
@@ -95,7 +95,7 @@ Investigate to cooperate with argo to support job dependencies.
 * Issue: https://github.com/volcano-sh/devices/issues/12
 * Owner: @peiniliu
 
-## v1.4 (To be updated)
+## v1.4
```

Comment:
When a release published, we'd also better history publish date for version.

Reply:
Resolved